app/models/ingredient.rb

```diff
@@ -1,3 +1,10 @@
 class Ingredient < ApplicationRecord
    validates_presence_of :name, :cost
+
+   has_many :recipe_ingredients
+   has_many :recipes, through: :recipe_ingredients
+
+   def self.alpha_ingredients
+      Ingredient.order(:name)
```

Comment:
```suggestion
      order(:name)
```
It is totally fine to explicitly say `Ingredient` here, but since this is a class method defined as `self.method_name`, you don't HAVE to. Just FYI.